docs-src/references/web-ui-server-env-vars.md

```diff
@@ -0,0 +1,173 @@
+
+Use `docker run` as the image name needed to configure the Web UI environmental variables.
+`docker run` requires at least 1 argument.
+
+`docker run [options] <image> <command> [args]`
+
+The Web UI Server is set up using specific environmental variables.
+These variables need to be passed when running the image.
+
+```
+docker run \
+    -e TEMPORAL_ADDRESS=127.0.0.1:7233 \
+    -e TEMPORAL_UI_PORT=8080 \
+    -e TEMPORAL_AUTH_ENABLED=true \
+    -e TEMPORAL_AUTH_PROVIDER_URL=https://accounts.google.com \
+    -e TEMPORAL_AUTH_CLIENT_ID=xxxxx-xxxx.apps.googleusercontent.com \
+    -e TEMPORAL_AUTH_CLIENT_SECRET=xxxxxxxxxxxxxxx \
+    -e TEMPORAL_AUTH_CALLBACK_URL=https://xxxx.com:8080/auth/sso/callback \
+    -e TEMPORAL_UI_ENABLED=true \
+    -e TEMPORAL_OPENAPI_ENABLED=true \
+    -e TEMPORAL_TLS_CA=../ca.cert \
+    -e TEMPORAL_TLS_CERT=../cluster.pem \
+    -e TEMPORAL_TLS_KEY=../cluster.key \
+    -e TEMPORAL_TLS_ENABLE_HOST_VERIFICATION=true \
+    -e TEMPORAL_TLS_SERVER_NAME=tls-server \
+    temporalio/ui:<tag>
+```
+
+Environmental variables are fields that are set to configure a server. The setup process involves the configuration of several components, including mTLS, authentication, and more.
+
+To view a full list of the environmental variables, refer to the [configuration template](https://github.com/temporalio/ui-server/blob/main/docker/config_template.yaml).
+
+The environmental variables needed to configure the WebUI server environment are explained below.
+Explanations and use cases are also provided for clarity.
+
+## `TEMPORAL_ADDRESS`
+
+The frontend address for the Temporal Server.
+
+`TEMPORAL_ADDRESS` allows the UI Server to [refresh as needed](/references/ui-configuration#refreshinterval) and [find the latest updates](/references/ui-configuration#notifyonnewversion) for the Temporal Server.
+
+This variable is required for the configuration of other environmental variables.
+`TEMPORAL_ADDRESS` is also needed when setting up [Cross-Origin Resource Sharing](/references/ui-configuration#cors).
+
+## `TEMPORAL_UI_PORT`
+
+The port used by the Temporal WebUI Server and the HTTP API.
+
+Defining this variable allows the Server and API to communicate effectively.
+
+This variable is needed for `TEMPORAL_OPENAPI_ENABLED` and all auth-related settings to work properly.
+`TEMPORAL_UI_ENABLED` requires a valid port number.
+
+## `TEMPORAL_AUTH_ENABLED`
+
+Enables or disables Temporal authentication and authorization methods.
+
+When enabled, the Temporal Server will use the provider information in the [UI configuration file](/references/ui-configuration#auth) to verify the identity of users.
+
+All auth-related environmental variables depend on `TEMPORAL_AUTH_ENABLED`.
+Disabling the variable will retain given values.
+
+All auth-related variables must be defined when this is enabled.
+
+## `TEMPORAL_AUTH_PROVIDER_URL`
+
+The URL for Temporal's [authentication and authorization OIDC provider](/references/ui-configuration#auth).
```

Comment:
```suggestion
The URL for Temporal's authentication and authorization OIDC provider.

This can be set as in the UI server configuration with [auth](/references/ui-configuration#auth).
```